src/Components/Hamburger/Hamburger.jsx

```diff
@@ -24,62 +26,33 @@ export const Hamburger = () => {
   return (
     <div>
       <ul className="hamburger-menu-container">
-        <NavLink to="/" className="hamburger-menu-item" style={getActiveStyle}>
-          <li className="hamburger-item flex-center">
-            <FaHome />
-            Home
-          </li>
-        </NavLink>
-        <NavLink
-          to="/videopage"
-          className="hamburger-menu-item"
-          style={getActiveStyle}
-        >
-          <li className="hamburger-item flex-center">
-            <MdExplore />
-            Explore
-          </li>
-        </NavLink>
-        <NavLink
-          to="/playlist"
-          className="hamburger-menu-item"
-          style={getActiveStyle}
-        >
-          <li className="hamburger-item flex-center">
-            <MdPlaylistAdd />
-            PlayList
-          </li>
-        </NavLink>
-        <NavLink
-          to="/likevideopage"
-          className="hamburger-menu-item"
-          style={getActiveStyle}
-        >
-          <li className="hamburger-item flex-center">
-            <AiOutlineLike />
-            Liked Videos
-          </li>
-        </NavLink>
-        <NavLink
-          to="/watchlater"
-          className="hamburger-menu-item"
-          style={getActiveStyle}
-        >
-          <li className="hamburger-item flex-center">
-            <FaBookmark />
-            Watch later
-          </li>
-        </NavLink>
-        <NavLink
-          to="/history"
-          className="hamburger-menu-item"
-          style={getActiveStyle}
-        >
-          <li className="hamburger-item flex-center">
-            <MdHistory />
-            History
-          </li>
-        </NavLink>
+        {sidebarOption.map(({ path, Icon, name }) => {
+          return (
+            <NavLink
+              to={`${path}`}
+              className="hamburger-menu-item"
+              style={getActiveStyle}
+              key={name}
+            >
+              <li className="hamburger-item flex-center">
+                <Icon.type />
+                {name}
+              </li>
+            </NavLink>
+          );
+        })}
+        {loginStatus ? (
+          <NavLink
+            to="/profilepage"
+            className="hamburger-menu-item"
+            style={getActiveStyle}
+          >
+            <li className="hamburger-item flex-center">
+              <FaUserCircle />
+              Profile
+            </li>
+          </NavLink>
+        ) : null}
```

Comment:
Can rewrite it as {loginStatus && <Profile Menu item>}
Why we are not including Profile item in sidebarOptions!!